### ISSUE TITLE
Type for slider and general navigation keyboard help content

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#483]
I noticed that we have a number of KeyboardHelpDialogs now that contain the SliderKeyboardHelpSection and the GeneralNavigationHelpSection. molecules-and-light, ohms-law, resistance-in-a-wire, and maybe there are others. We should have a general type that assemples these sections for us.

**Comment by jessegreenberg** [comment]
Noticed during https://github.com/phetsims/scenery-phet/issues/478.

**Comment by jessegreenberg** [comment]
OK, I made this, and it can be used currently by molecules-and-light, ohms-law, plinko-probability, resistance-in-a-wire. For these cases, this Dialog can be used directly by {{sim}}-main `keyboardHelpNode` option. Then all of the *KeyboardHelpContent files for molecules-and-light, ohms-law, plinko-probabiliy, and resistance-in-a-wirie can be deleted. I think this is the right way forward, @zepumph any objections?

**Comment by zepumph** [comment]
This looks really nice! Thanks.

**Comment by jessegreenberg** [comment]
OK, thanks @zepumph sounds good.

**Comment by jessegreenberg** [comment]
Done for the sims mentioned in https://github.com/phetsims/scenery-phet/issues/483#issuecomment-480550021. Closing.
